src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerGetAdminMediasByIdIntTest.java

```diff
@@ -271,6 +273,65 @@ void shouldDenyAccess(SecurityRoleEnum role) throws Exception {
                                     """, jsonString, JSONCompareMode.STRICT);
     }
 
+    @Test
+    void getMediaById_shouldReturnCaseWithCasIdAndActualHearing_whenMediaHasLinkedCase() throws Exception {
```

Comment:
Suggest including tests cases to verify the newly added `hearings` attributes are returned as expected.

Reply:
agree - have added hearings